docs/iceberg-rest-service.md

```diff
@@ -13,21 +13,29 @@ The Gravitino Iceberg REST Server follows the [Apache Iceberg REST API specifica
 
 ### Capabilities
 
-- Supports the Apache Iceberg REST API defined in Iceberg 1.3.1, supports all namespace and table interfaces. `Token`, `ReportMetrics`, and `Config` interfaces aren't supported yet.
-- Works as a catalog proxy, supporting `HiveCatalog` and `JDBCCatalog`.
+- Supports the Apache Iceberg REST API defined in Iceberg 1.3.1, supports all namespace and table interfaces. `Token`, `ReportMetrics`, and `Config` interfaces aren't supports yet.
```

Comment:
", and supports all namespace"
aren't supported yet.

Reply:
done